packages/airnode-deployer/terraform/airnode/aws/main.tf

```diff
@@ -2,41 +2,11 @@
 # * switch between local and remote lambda source
 # * variable validation
 
-module "initializeProvider" {
+module "run" {
   source = "./modules/function"
 
-  name               = "${local.name_prefix}-initializeProvider"
-  handler            = "index.initializeProvider"
-  source_dir         = var.handler_dir
-  memory_size        = 768
-  timeout            = 17
-  configuration_file = var.configuration_file
-  secrets_file       = var.secrets_file
-  environment_variables = {
-    HTTP_GATEWAY_URL = var.api_key == null ? null : "${module.testApiGateway[0].api_url}/test"
-  }
-}
-
-module "callApi" {
-  source = "./modules/function"
-
-  name               = "${local.name_prefix}-callApi"
-  handler            = "index.callApi"
-  source_dir         = var.handler_dir
-  timeout            = 10
-  configuration_file = var.configuration_file
-  secrets_file       = var.secrets_file
-  memory_size        = 256
-  environment_variables = {
-    HTTP_GATEWAY_URL = var.api_key == null ? null : "${module.testApiGateway[0].api_url}/test"
-  }
-}
-
-module "processProviderRequests" {
-  source = "./modules/function"
-
-  name               = "${local.name_prefix}-processProviderRequests"
-  handler            = "index.processProviderRequests"
+  name               = "${local.name_prefix}-run"
+  handler            = "index.run"
   source_dir         = var.handler_dir
   memory_size        = 768
   timeout            = 32
```

Comment:
Does this mean they all have the same timeout?

Reply:
Unfortunately, yes. I used the highest timeout of those three handlers. But as far as I know, we still have a "manual" timeout in the code, so, it shouldn't be a problem.

Reply:
Ok cool. We'll probably want to test this on both cloud providers then

Reply:
I think @aquarat tested that when he was doing stress tests. Or do you mean writing additional tests for that?

Reply:
Actually I think that should be fine - don't think we need additional tests. It's nice having the timeout in the function because then it means that there should just be a single place where the values need to be updated.